### PR TITLE
Update _colors.native.scss

### PR DESCRIPTION
**Pull request by guarani**
## Description

Block lines (e.g. borders, separators, etc) and fills (e.g. icon fills)
- `gray-lighten-20` becomes `neutral-quaternary-light`
- `gray-70` (and in some cases `gray-50`) become `neutral-secondary-dark` 

## How has this been tested?

View screenshots below or build and run on device

## Screenshots 

| | Before | After |
| - | - | - |
| Light Mode | Screen Shot 2020-09-17 at 12 05 22 | Screen Shot 2020-09-17 at 11 57 55 |
| Dark Mode | Screen Shot 2020-09-17 at 12 05 31 | Screen Shot 2020-09-17 at 11 58 19 |

## Types of changes
- New color palette changes

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/base-styles/_colors.native.scss [hunk 1]
```diff
@@ -105,7 +105,7 @@ $gray-text-min: darken($gray, 18%); //#537994
 
 // Shades of gray
 $gray-lighten-10: lighten($gray, 10%); // #a8bece
-$gray-lighten-20: lighten($gray, 20%); // #c8d7e1
+$studio-gray-10: #c3c4c7;
```
Comment (enejb) [inline packages/base-styles/_colors.native.scss:108]: I think we should also add design token type variable. 

So something like:
```suggestion
$studio-gray-10: #c3c4c7;

$block-line-color: $studio-gray-10;
$icon-uploading-fill: $studio-gray-10;
```

File: packages/block-library/src/more/editor.native.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // @format
 
 .moreLine {
-	background-color: $gray-lighten-20;
+	background-color: $studio-gray-10;
```
Comment (enejb) [inline packages/block-library/src/more/editor.native.scss:4]:
```suggestion
	background-color: $block-line-color;
```

File: packages/block-library/src/nextpage/editor.native.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // @format
 
 .nextpageLine {
-	background-color: $gray-lighten-20;
+	background-color: $studio-gray-10;
```
Comment (enejb) [inline packages/block-library/src/nextpage/editor.native.scss:4]:
```suggestion
	background-color: $block-line-color;
```

File: packages/block-library/src/pullquote/figure.native.scss [hunk 4]
```diff
@@ -5,8 +5,8 @@
 
 .light {
 	@extend %shared;
-	border-top-color: $gray-lighten-20;
-	border-bottom-color: $gray-lighten-20;
+	border-top-color: $studio-gray-10;
```
Comment (enejb) [inline packages/block-library/src/pullquote/figure.native.scss:8]:
```suggestion
	border-top-color: $block-line-color;
```

File: packages/block-library/src/pullquote/figure.native.scss [hunk 5]
```diff
@@ -5,8 +5,8 @@
...
+	border-bottom-color: $studio-gray-10;
```
Comment (enejb) [inline packages/block-library/src/pullquote/figure.native.scss:9]:
```suggestion
	border-bottom-color: $block-line-color;
```

File: packages/block-library/src/video/style.native.scss [hunk 6]
```diff
@@ -69,7 +69,7 @@
 }
 
 .iconUploading {
-	fill: $gray-lighten-20;
+	fill: $studio-gray-10;
```
Comment (enejb) [inline packages/block-library/src/video/style.native.scss:72]:
```suggestion
	fill: $icon-uploading-fill;
```

File: packages/components/src/mobile/image/style.native.scss [hunk 7]
```diff
@@ -41,7 +41,7 @@
 }
 
 .iconUpload {
-	fill: $gray-lighten-20;
+	fill: $studio-gray-10;
```
Comment (enejb) [inline packages/components/src/mobile/image/style.native.scss:44]:
```suggestion
	fill: $icon-uploading-fill;
```

File: packages/base-styles/_colors.native.scss (hunk 1 above)
Comment (iamthomasbishop) [reply packages/base-styles/_colors.native.scss:108]: @enejb @Guarani Any chance we can skip the Studio grays and translate the old grays to our neutral grays (in the above case, `gray-lighten-20` would become our "tertiary" neutral)?

Comment (iamthomasbishop) [reply packages/base-styles/_colors.native.scss:108]: To clarify, we'd keep the base declarations for the studio grays in the project, but for most elements, the neutral grays would be preferred.

Comment (guarani) [reply packages/base-styles/_colors.native.scss]: SGTM 🙂

**Comment by guarani** [comment]
I still need to add screenshots above for how the icon fills look.

**Comment by iamthomasbishop** [comment]
> gray-lighten-20 becomes neutral-tertiary-light

Apologies, I might have provided the wrong target color — I think `gray-lighten-20` should translate to `neutral-quaternary-`.

image

> gray-70 (and in some cases gray-50) become neutral-tertiary-dark

Similarly, I think both `gray-70` and `gray-50` should probably use `neutral-secondary`.

image

I'll take a look at the sheet we're referencing to make sure it is accurate—sorry for the confusion!

**Comment by iamthomasbishop** [comment]
@guarani And one more note regarding icons: as a general rule of thumb, icon fills (in static state) are always going to be `neutral-secondary`.

**Comment by guarani** [comment]
Closing for now since I'm not actively working on this and it might have become stale.
It can be re-opened if or when work continues to update these colors.

**Pull request closed without merging**